packages/packages/api/src/medical/client/metriport.ts

```diff
@@ -353,6 +353,20 @@ export class MetriportMedicalApi {
     return documentQuerySchema.parse(resp.data);
   }
 
+  /**
+   * Returns the document query status for the specified patient.
+   *
+   * @param patientId Patient ID for which to retrieve document query status.
+   * @return The document query progress & status indicating whether its being executed or not.
+   */
+  async getDocumentQuery(patientId: string): Promise<DocumentQuery> {
```

Comment:
nit: should we call this `getDocumentQueryStatus`? We're returning the status - not the query.